kirby/api/__init__.py

```diff
@@ -0,0 +1,46 @@
+import requests
+
+from .context import ContextManager, ctx
+
+
+class Kirby:
+    def __init__(self, env_signature, session=None):
+        ctx_manager = ContextManager(env_signature)
+        ctx_manager.load()
+        self.ctx = ctx
+        self._session = session or requests.session()
+
+        self._register()
+
+    def get_topic_id(self, topic_name):
+        result = self._session.get(
+            "/".join([self.ctx.KIRBY_WEB_SERVER, "topic"]),
+            params={"name": topic_name},
+        )
+        assert result.status_code == 200
+        return result.json()["id"]
+
+    def _register(self, params=None):
```

Comment:
If params can only contain two keys, why not explicitly declare 2 parameters: `source_id` and `destination_id` ?

Reply:
Indeed, it seems better. What if the endpoint has to change?

Reply:
then you pass extra arguments?

Reply:
The keys are now used as parameters